docs/schema/codelists.md

```diff
@@ -23,7 +23,7 @@ Codes are case-sensitive, and are generally provided as English language camelCa
 
 ### Release Tag
 
-A contracting process can result in a number of releases of information over time. These must be tagged to indicate the stage of the contracting process they relate to.
+A contracting process can result in a number of releases of information over time. These must be tagged to indicate the stage of the contracting process they relate to. Additional codes can be used based on the publisher's use cases, such as the mapping of notice or event types.
```

Comment:
Make the new text a separate paragraph, to separate the MUST from the MAY.

Also, refer to user needs, not publisher's use cases.

```suggestion
A contracting process can result in a number of releases of information over time. These must be tagged to indicate the stage of the contracting process they relate to.

Additional codes may be used to label releases, based on user needs: for example, to indicate the notice or form to which a release corresponds.
```